source/diet/html.d

```diff
@@ -206,14 +291,49 @@ string getHTMLMixin(in Document doc, string range_name = dietOutputRangeName, HT
 	CTX ctx;
 	ctx.pretty = style == HTMLOutputStyle.pretty;
 	ctx.rangeName = range_name;
-	string ret = "import diet.internal.html : htmlEscape, htmlAttribEscape;\n";
+	string ret = "import diet.internal.html : htmlEscape, htmlAttribEscape, filterHTMLAttribEscape;\n";
 	ret ~= "import std.format : formattedWrite;\n";
 	foreach (i, n; doc.nodes)
 		ret ~= ctx.getHTMLMixin(n, false);
 	ret ~= ctx.flushRawText();
 	return ret;
 }
 
+/**
+  This returns only the NON-code portions of the diet template. The return value is a concatenated string with each string of HTML code separated by a null character. To extract the strings to send into the live renderer, split the string based on a null character.
+  */
+string getHTMLRawTextOnly(in Document doc, string range_name = dietOutputRangeName, HTMLOutputStyle style = HTMLOutputStyle.compact) @safe
```

Comment:
I need them public (including `getHTMLMixin` which is already public) because I call them in my diet pre-compiler project (I'm going to finalize and release that as soon as this is merged). See here: https://github.com/schveiguy/dietpc/blob/97d0812ddbf03139fcca1f1ede6107093148c2e6/source/app.d#L71-L78